### PR TITLE
Profile Page

### PR DESCRIPTION
**Pull request by trowland10**
- Create a page to view other users' profiles. 
- Displays The User's library. 
- Fixed the country dropdown selector issue in "update account"

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -31,6 +31,16 @@ handlebars.handlebars.registerHelper('ifEquals', function(arg1, arg2, options){
     return (arg1 == arg2) ? options.fn(this) : options.inverse(this);
 })
 
+handlebars.handlebars.registerHelper("select", function(value, options) {
+    return options.fn(this)
+      .split('\n')
```
Comment (mkoeplin) [inline app.js:36]: This is a great use of registering a custom handlebars function!

File: routes/index.js [hunk 2]
```diff
@@ -376,4 +398,51 @@ router.get('/getAddress/(:id)', common.isAuthenticated, (req, res, next) => {
...
 
+router.get('/profile/:userId', common.isAuthenticated, (req, res, next) => {
+    var userId = req.params.userId;
+    var payload = {};
+    payload.user;
+    var library = [];
+    var sentBooks = 0;
+    var receivedBooks = 0;
+
+    sql.pool.query(selectUserinfo, [userId], (err, result) => {
```
Comment (mkoeplin) [inline routes/index.js:409]: You might want to handle the errors consistently through this function and switch these first two potential error points to use the flash and redirect as well. Not a deal breaker but just so the end user would know what is going on in the case these errors are hit

**Review by mkoeplin — APPROVED** [review]
Overall, great job! This pull request really shows how much you've learned since the beginning of this class so that is awesome to see. I just left one comment regarding error handling but it's more of a suggestion not a requirement

**Comment by jung8** [comment]
Looks good!

**Pull request merged**